cmds/cpu/cpu.go

```diff
@@ -203,16 +204,26 @@ func runClient(host, a, port, key string) error {
 	return nil
 }
 
-func env(s *ossh.Session, envs ...string) {
+func env(s *ossh.Session, envs ...string) error {
 	for _, v := range append(os.Environ(), envs...) {
 		env := strings.SplitN(v, "=", 2)
 		if len(env) == 1 {
 			env = append(env, "")
 		}
 		if err := s.Setenv(env[0], env[1]); err != nil {
-			log.Printf("Warning: s.Setenv(%q, %q): %v", v, os.Getenv(v), err)
+			return fmt.Errorf("Warning: s.Setenv(%q, %q): %v", v, os.Getenv(v), err)
```

Comment:
Should this change be reverted? It looks like a leftover from the env var approach.

Reply:
Huh, now I get it... the env var is abused for marshalling. Was a bit unintuitive first. 😅

Reply:
This also passes all environment variables.

Reply:
hence os.Environ()